scm/src/scm_input.F90

```diff
@@ -2138,7 +2195,9 @@ subroutine get_case_init_DEPHY(scm_state, scm_input)
     else
       scm_input%input_lh_flux_sfc = input_force_sfc_lat_flx(:)
     end if
-  else if (trim(input_surfaceForcingLSM) == 'lsm') then
+  end if
+  
+  if (trim(input_surfaceForcingLSM) == 'lsm') then
```

Comment:
Note to self: This is needed to not have LSM ICs be mutually exclusive to surfaceForcingT == 'ts'. For the MOSAiC cases, we have a forcing surface T _and_ LSM IC data for the sea ice. This allows us to read in LSM IC data and force the surface temperature, like the case wants. This does **not** break the GABLS3 cases (tested).